src/components/views/Login.js

```diff
@@ -72,7 +73,39 @@ const Login = () => {
     }
   }
 
+  const getGeoLocation = async (userId) => {
+
+    // how to handle if we get user location
+    const successCallback = (position) => {
+      const latitude = position.coords.latitude;
+      const longitude = position.coords.longitude;
+      setLocation([latitude, longitude]);
+    };
+
+    // how to handle if user denies access to location
+    const errorCallback = (error) => {
+      console.log(`Error: ${error.code}`);
+      console.log(`Error Message: ${error.message}`);
+      window.alert("Your location could not be determined. You were set to the default location (0\"N, 0\"E).");
+    };
+
+    // if geolocation is not supported by the browser, do so
+    if (!navigator.geolocation) {
+      window.alert("Geolocation is not supported by your browser. You were set to the default location (0\"N, 0\"E).");
```

Comment:
consider trying getting the last known location of a user and not setting to 0/0. As it is possible that the user registered and used the app on a different browser, where geolocation was supported and now uses another browser. I think it would be more sensible first checking if the user already has an "old" geolocation (long + lat) saved and setting it to that and then only if the user doesnt have that setting it to 0 0

Reply:
good point, i'll still return [0,0] from the client to the server if no location is accessable, but will check on the server if there is a non-default location available, and if so AND [0,0] is sent as update, then i won't update.